tests/unit/matrix/test_measure.py

```diff
@@ -1371,3 +1413,491 @@ def but_the_subtotal_blocks_are_NaNs_when_an_MR_dimension_is_present(
     @pytest.fixture
     def dimensions_(self, request):
         return instance_mock(request, Dimension), instance_mock(request, Dimension)
+        return (instance_mock(request, Dimension), instance_mock(request, Dimension))
+
+
+# === Marginals ===
+
+
+class Describe_BaseSecondOrderMarginal(object):
+    """Unit test suite for `cr.cube.matrix.measure._BaseSecondOrderMarginal` object."""
+
+    def it_has_default_value_for_is_defined(self):
+        marginal = _BaseSecondOrderMarginal(None, None, None, None)
+        assert marginal.is_defined is True
+
+    @pytest.mark.parametrize("orientation", (OR.ROWS, OR.COLUMNS))
+    def it_provides_orientation(self, orientation):
+        marginal = _BaseSecondOrderMarginal(None, None, None, orientation)
+
+        assert marginal.orientation == orientation
+
+    @pytest.mark.parametrize(
+        "orientation, array, expected",
+        (
+            (OR.ROWS, np.array([[0, 1, 2], [3, 4, 5]]), [3, 12]),
+            (OR.COLUMNS, np.array([[0, 1, 2], [3, 4, 5]]), [3, 5, 7]),
+            (OR.ROWS, np.array([]).reshape(0, 6), []),
+            (OR.COLUMNS, np.array([]).reshape(6, 0), []),
+        ),
+    )
+    def it_can_apply_along_orientation(self, orientation, array, expected):
```

Comment:
A `_to_help` suffix wouldn't be out of place here, to indicate this is testing a helper and not an interface method.